### ISSUE TITLE
kd loss question

### ISSUE THREAD

**Issue by ninisy** [philschmid/knowledge-distillation-transformers-pytorch-sagemaker#1]
loss_logits = (loss_function(
            F.log_softmax(outputs_student.logits / self.args.temperature, dim=-1),
            F.softmax(outputs_teacher.logits / self.args.temperature, dim=-1)) * (self.args.temperature ** 2))
why loss_function is multiplied by  (self.args.temperature ** 2)?
